editor.planx.uk/src/pages/FlowEditor/components/Flow/FeedbackLog/FeedbackLog.tsx

```diff
@@ -17,12 +17,23 @@ import {
 import ErrorSummary from "ui/shared/ErrorSummary/ErrorSummary";
 
 import { ExpandableHelpText } from "./components/ExpandableHelpText";
-import { feedbackTypeOptions } from "./feedbackFilterOptions";
+import { StatusChip } from "./components/StatusChip";
+import { feedbackTypeOptions, statusOptions } from "./feedbackFilterOptions";
 import { FeedbackLogProps } from "./types";
 import { EmojiRating, feedbackTypeText, stripHTMLTags } from "./utils";
 
 export const FeedbackLog: React.FC<FeedbackLogProps> = ({ feedback }) => {
   const columns: ColumnConfig<Feedback>[] = [
+    {
+      field: "status",
+      headerName: "Status",
+      type: ColumnFilterType.ARRAY,
+      customComponent: StatusChip,
+      columnOptions: {
+        valueOptions: statusOptions,
+        filterable: false,
```

Comment:
This is temporarily set to `false` as I need to get my head around the changes here - https://github.com/theopensystemslab/planx-new/pull/4453

It should be the case that one a column is set to `singleSelect` it automatically gets treated as an enum but the above PR undoes this default behaviour. I'll dig into this a bit more tomorrow and may have a few follow up questions!

Reply:
I've got an upcoming PR which will address this, leaving as non-filterable initially.